src/pages/ClubList/ClubList.scss

```diff
@@ -1,14 +1,11 @@
-.get-involved {
-  &__description {
-    margin-bottom: 50px;
-  }
-
+.clublist {
```

Comment:
This should be `clubs` its more common to use plural to describe the listing page